client/src/partials/templates/modals/findEntity.modal.html

```diff
@@ -1,69 +1,79 @@
-<div class="modal-header">
-  <h4>
-    <i class="glyphicon glyphicon-search"></i>
-    {{ 'VOUCHERS.COMPLEX.ENTITY' | translate }}
-    <span
-      ng-if="$ctrl.result.uuid"
-      ng-click="$ctrl.refresh()"
-      class="btn btn-default pull-right">
-      <i class="glyphicon glyphicon-refresh"></i>
-    </span>
-  </h4>
-</div>
+<ng-form
+  name="EntityForm"
+  autocomplete="off"
+  autocorrect="off"
+  autocapitalize="off"
+  data-modal="entity"
+  novalidate>
 
-<div class="modal-body clearfix">
+  <div class="modal-header">
+    <h4>
+      <span class="glyphicon glyphicon-search"></span>
+      {{ 'VOUCHERS.COMPLEX.ENTITY' | translate }}
 
-  <div class="form-group">
+      <span
+        ng-show="$ctrl.result.uuid"
+        ng-click="$ctrl.refresh()"
+        class="btn btn-default btn-sm pull-right">
```

Comment:
I thought it was mostly preexisting - it has no effect on the merge. 

As I understand it, it would have to be on the container. Potentially using a div that takes up the rest of the space and uses `display : inline-block` or something.